GEOSldas_App/util/postproc/ObsFcstAna_stats/Main_example.py

```diff
@@ -0,0 +1,407 @@
+#!/usr/bin/env python3
+
+"""
+Sample script for computing/plotting data assimilation (DA) diagnostics from 
+GEOSldas ObsFcstAna output.
+
+Processing workflow:
+1. User provides experiment information and calls postproc_ObsFcstAna to:
+   - Calculate/save monthly sums of Obs/Fcst/Ana and squared/products
+   - Calculate mean/stdv of Obs/Fcst/Ana
+2. User computes desired DA diagnostic statistics and creates plots
+
+Usage on Discover:
+    $ module load python/GEOSpyD
+    $ ./Main_example.py
+    
+    # Background execution:
+    $ nohup ./Main_example.py > out.log &
+
+Author: Q. Liu
+Last Modified: Apr., 2025
+"""
+
+import sys;       sys.path.append('../../shared/python/')
+import warnings;  warnings.filterwarnings("ignore")
+import os
+
+import numpy             as np
+import matplotlib.pyplot as plt
+
+from datetime               import datetime, timedelta
+from dateutil.relativedelta import relativedelta
+from netCDF4                import Dataset, num2date
+from mpl_toolkits.basemap   import Basemap
+
+from read_GEOSldas          import read_tilecoord, read_obs_param
+from util                   import make_folder, array2grid
+from plot                   import plotMap
+from EASEv2                 import EASEv2_ind2latlon
+
+from postproc_ObsFcstAna    import postproc_ObsFcstAna
+
+# Uncomment if to run the script in the background to see the standard output while running 
+# import io
+#sys.stdout = io.TextIOWrapper(open(sys.stdout.fileno(), 'wb', 0), write_through=True)
+#sys.stderr = io.TextIOWrapper(open(sys.stderr.fileno(), 'wb', 0), write_through=True)
+
+# Define time range for processing (Year, Month, Day)
+start_time = datetime(2015,4,1)
+end_time   = datetime(2016,4,1)
+
+# -------------------------------- Experiments Information -----------------------------------------
+# Supports single experiment or multiple experiments.
+# All experiments must have identical tilecoords and number/order of observation species.
+# If the default "species" number/order do not match, need to set the *optional*
+#   "select_species" key to get a match, i.e. same species sequences.
```

Comment:
I'm a bit confused by these two seemingly contradictory comments.  The 2nd comment seems to suggest it is possible for multiple experiments to **not** have the same number/order of obs species

Reply:
experiments don't need to have the same species sequences, but the select_species should result in the same species sequences. I will think of better wording, but feel free to edit.